documentation/specs/src/economics/inflation-system.md

```diff
@@ -1,119 +1,80 @@
-
-We define $I_{PoS}$ as a PD controller as follows. 
-
-$$A(t)=K_1(R(t)-R_{target})+K_2(\frac{dR}{dt})$$
-
-If $I_{PoS}^{min}< I_{PoS}< I_{PoS}^{max}$ then $\frac{dI_{PoS}}{dt}=A(t)$.
-
-If $I_{PoS}^{min}< I_{PoS}$ then $\frac{dI_{PoS}}{dt}=max(A(t),0)$.
-
-If $I_{PoS}< I_{PoS}^{max}$ then $\frac{dI_{PoS}}{dt}=min(A(t),0)$.
-
-For $I_{PoS}^{min}=0.05$, $I_{PoS}^{max}=0.15$, $I_{PoS,target}=0.10$, and $R_{target}=0.50$ we set $K_1=-0.01$ and $K_2=-0.2$. Lets review what these parameters give us with examples as follows. 
-
-**Example 1:** If $I= I_{PoS,target}=0.10$ and $R_{target}=0.50$, but then $R$ drops quickly to $0.25$, then the effect of the $K_2$ term will be to increase $I_{PoS}$ by $-0.2 \times -0.25=0.05$ and inflation will hit its maximum value of $0.15$. Changes in $R$ smaller than $0.25$ will not cause inflation to hit its maximum or minimum quickly.
-
-**Example 2:** If $I_{PoS}=0.05$, but $R$ holds steady at $0.40$, then $K_1$ term will cause $I$ to increase by $-0.01 \times -0.10=0.001$ per day/epoch. $I_{PoS}$ will take 100 days to reach its maximum. This is slow compared to the unbonding period, allowing delegators time to react.
+### Shielded pool rewards
 
+Privacy provided by the MASP in practice depends on how many users use the shielded pool and what assets they use it with. To increase the likelihood of a sizeable privacy set, Namada pays a variable portion of inflation, up to 10% per annum, to shielded pool incentives, which are allocated on a per-asset basis by a PD-controller targeting specific amounts of each asset being locked in the shielded pool. See [shielded pool incentives](./shielded-pool-incentives.md) for details.
 
----
+### Public goods funding
 
-We define $I_{L}$ as a PD controller follows. 
+Namada provides 10% per annum inflation for other non-algorithmically-measurable public goods. See [public goods funding](./public-goods-funding.md) for details.
 
-$$A(t)=K_1(L(t)-L_{target})+K_2(\frac{dL}{dt})$$
+## Detailed inflation calculation model
 
-If $I_{L}^{min}< I_{L}< I_{L}^{max}$ then $\frac{dI_{L}}{dt}=A(t)$.
+Inflation is calculated and paid per-epoch as follows.
 
-If $I_{L}^{min}< I_{L}$ then $\frac{dI_{L}}{dt}=max(A(t),0)$.
+First, we start with the following fixed (governance-alterable) parameters:
 
-If $I_{L}< I_{L}^{max}$ then $\frac{dI_{L}}{dt}=min(A(t),0)$.
+- $Cap_{PoS}$ is the cap of proof-of-stake reward rate, in units of percent per annum
+- $Cap_{SP-A}$ is the cap of shielded pool reward rate for each asset $A$, in units of percent per annum
+- $R_{PGF}$ is the public goods funding reward rate, in units of percent per annum
+- $R_{PoS-Target}$ is the target staking ratio (genesis default 2/3)
+- $R_{SP-A-Target}$ is the target amount of asset $A$ locked in the shielded pool (separate value for each asset $A$)
+- $EpochsPerYear$ is the number of epochs per year (genesis default 365)
```

Comment:
we currently have these params https://github.com/anoma/namada/blob/958715216c93d76b29b59f4765c38a78c4d119df/shared/src/ledger/parameters/mod.rs#L141 
Should we use `min_duration` to estimate `EpochsPerYear`? We should probably make sure that `min_duration > min_num_of_blocks * consensus_timeout_commit` generously enough so that we're not waiting for blocks to switch epoch

Reply:
Yes, we should use `min_duration` to estimate `EpochsPerYear` (but the latter should still be constant)